.vscode/settings.json

```diff
@@ -9,4 +9,5 @@
     "go.testEnvFile": "${workspaceFolder}/shared_files/decrypted/local.env",
     "go.testTimeout": "2m",
     "go.buildOnSave": "package",
+    "go.lintTool": "golangci-lint"
```

Comment:
so you don't freeze the editor, golangci-lint recommends that you add:
 `"go.lintFlags": [
  "--fast"
]`